### PR TITLE
Format doc footer with copyright notice

### PR DESCRIPTION
**Pull request by gautierronan**

### CODE REVIEW

**Review by abocquet — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -34,6 +34,14 @@ theme:
       toggle:
         icon: material/brightness-4
         name: Switch to light mode
+copyright: Copyright &copy 2023 dynamiqs
+extra:
+  generator: false
```
Comment (pierreguilmin) [inline mkdocs.yml:39]: Mmmm, have you read the "Please read this before removing the generator notice" part here > https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/?h=#generator-notice? I think we should either sponsor mkdocs-material, either keep the notice.

File: mkdocs.yml [hunk 2]
```diff
@@ -34,6 +34,14 @@ theme:
       toggle:
         icon: material/brightness-4
         name: Switch to light mode
+copyright: Copyright &copy 2023 dynamiqs
```
Comment (pierreguilmin) [inline mkdocs.yml:37]:
```suggestion
copyright: "&copy Copyright 2023, dynamiqs developers."
```

Comment (pierreguilmin) [reply mkdocs.yml:37]: To mimic numpy and pytorch docs.

File: mkdocs.yml [hunk 3]
```diff
@@ -34,6 +34,14 @@ theme:
...
+extra:
+  generator: false
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/dynamiqs/dynamiqs
+    - icon: fontawesome/brands/slack
+      link: https://dynamiqs-org.slack.com/
```
Comment (pierreguilmin) [inline mkdocs.yml:44]: We should put the invite link (it redirects automatically if you signed up on the Slack)

https://join.slack.com/t/dynamiqs-org/shared_invite/zt-1z4mw08mo-qDLoNx19JBRtKzXlmlFYLA

**Review by pierreguilmin — CHANGES_REQUESTED** [review]
Nice!

**Review by pierreguilmin — APPROVED** [review]

**Pull request merged**